### ISSUE TITLE
Unable to parse loglevel

### ISSUE THREAD

**Issue by faust64** [openshift/elasticsearch-operator#204]
Starting the ElasticSearch operator on OpenShift, we would find the following warning:

```
time="2019-11-21T15:05:18Z" level=warning msg="Unable to parse loglevel \"\""
```

Source image for that Pod: registry.redhat.io/openshift4/ose-elasticsearch-operator@sha256:69968bc98d29773b89ba9b3c3c5b1fc44d8df5226934683e3029ae14f4829fab

Is there anything I should do?

**Comment by jcantrill** [comment]
fixed by #279

**Comment by faust64** [comment]
just checking, I'll trust you on that one.
